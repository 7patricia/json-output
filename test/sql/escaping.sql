\set ECHO none
SET client_min_messages TO error;
\t on
\x off
SELECT 'created logical replication slot'
  FROM pg_create_logical_replication_slot('json', 'json-output');

BEGIN;

CREATE SCHEMA IF NOT EXISTS json;
SET LOCAL search_path TO json, public;
CREATE EXTENSION IF NOT EXISTS hstore;

CREATE TABLE IF NOT EXISTS "****" (
  i   integer PRIMARY KEY
);

INSERT INTO "****" (i) SELECT i FROM generate_series(1, 8) AS i;

CREATE VIEW changedata AS
SELECT data FROM pg_logical_slot_get_changes('json', NULL, NULL);

END;

SELECT * FROM json.changedata;

SELECT 'deleted logical replication slot';

DROP SCHEMA json CASCADE;
