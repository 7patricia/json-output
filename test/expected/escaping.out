\set ECHO none
 created logical replication slot

 {"schema":"json","table":"****","action":"insert","new":{"i":1}}
 {"schema":"json","table":"****","action":"insert","new":{"i":2}}
 {"schema":"json","table":"****","action":"insert","new":{"i":3}}
 {"schema":"json","table":"****","action":"insert","new":{"i":4}}
 {"schema":"json","table":"****","action":"insert","new":{"i":5}}
 {"schema":"json","table":"****","action":"insert","new":{"i":6}}
 {"schema":"json","table":"****","action":"insert","new":{"i":7}}
 {"schema":"json","table":"****","action":"insert","new":{"i":8}}

 deleted logical replication slot

